### PR TITLE
Btm 84 filter lambda tests

### PR DESCRIPTION
**Pull request by NKannan14**
Refactored tests to use aws sdk javascript v3 and added clients, helper classes
added jest test integration config file
updated readme to run the integration tests

### CODE REVIEW

File: integration_tests/helpers/cloudWatchHelper.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  const latestLogStearmResponse: LogStream[] = response.logStreams ?? [];
+  return latestLogStearmResponse;
+}
+
+async function getCloudWatchLatestLogStreamName() {
+  const logStream: LogStream[] = await getCloudWatchLatestLogStreams();
+  if (logStream.length > 0) {
+    const result = logStream[0].logStreamName as string;
+    return result;
+  }
```
Comment (markschnitziusgovuk) [inline integration_tests/helpers/cloudWatchHelper.ts:37]: Should this throw/fail in the 'else' clause?  What happens if undefined is returned?

File: integration_tests/helpers/cloudWatchHelper.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  };
+  console.log("Filtered parameters:", params);
+  const response: FilterLogEventsCommandOutput =
+    await cloudWatchLogsClient.send(new FilterLogEventsCommand(params));
+  console.log("FilteredCloudWatchLog:", response);
+  const events: FilteredLogEvent[] = response.events ?? [];
+  if (events.length > 0) {
+    return events;
+  }
+  console.error("Filetered events empty", events);
```
Comment (markschnitziusgovuk) [inline integration_tests/helpers/cloudWatchHelper.ts:57]: Nitpick: fix spelling of 'filtered'.

**Review by markschnitziusgovuk — CHANGES_REQUESTED** [review]

File: integration_tests/helpers/cloudWatchHelper.ts (hunk 1 above)
Comment (NKannan14) [reply integration_tests/helpers/cloudWatchHelper.ts:39]: thanks added else clause

File: integration_tests/helpers/cloudWatchHelper.ts (hunk 2 above)
Comment (NKannan14) [reply integration_tests/helpers/cloudWatchHelper.ts]: fixed

**Review by markschnitziusgovuk — APPROVED** [review]

**Pull request merged**